core/arch/arm/plat-hikey/platform_config.h

```diff
@@ -38,10 +38,10 @@
 #endif /* ARM64 */
 
 /* PL011 UART */
-#if defined(CFG_HIKEY_UART3)
-#define CONSOLE_UART_BASE	0xF7113000
-#else
+#if defined(CFG_HIKEY_UART0)
```

Comment:
That's OK but I'd rather use a name that could be re-used by other platforms. Such as `CFG_CONSOLE_UART`.
Something like this:
```
#if defined(CFG_CONSOLE_UART) && (CFG_CONSOLE_UART == 0)
#define CONSOLE_UART_BASE       0xF8015000
#elif !defined(CFG_CONSOLE_UART) || (CFG_CONSOLE_UART == 3)
#define CONSOLE_UART_BASE       0xF7113000
#else
#error Unknown console UART
#endif
```